BUILD_DETAIL.md

```diff
@@ -40,7 +40,7 @@ The spec suite performs a couple extra checks that are worth noting:
   available for use in any context, we want to minimize how many bits of the standard
   library we load and use. Otherwise, RSpec's use of part of the standard library could
   mask a problem where a gem author forgets to load a part of the standard library they
-  rely on. The spec suite contains a spec that defines a whitelist of allowed loaded
+  rely on. The spec suite contains a spec that defines a allowlist of allowed loaded
```

Comment:
`defines a allowlist of allowed` is a bit duplicative, can you change it to just `defines a list of allowed`